Dockerfile

```diff
@@ -1 +1,7 @@
-FROM google/nodejs-runtime
+FROM node:14
```

Comment:
not: what about `node:15-slim` ?

Reply:
Node.js tends to use even numbers for LTS releases, and odd for experimental preview releases that will never be supported.  Happy to use slim, but let's keep this back on 14 since it's LTS.

Reply:
oh yes, you're right I forgot about that!